types/apollo-upload-client/tsconfig.json

```diff
@@ -16,7 +16,11 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "paths":{
```

Comment:
I don't think you want this. You've added `@apollo/core` to the list of allowed packages, so you just need to `npm i` in this directory to get it working locally.

Reply:
I haven't added `@apollo/core` anywhere? Do you mean `@apollo/client`?

Reply:
sorry, yes. typo!